src/index.css

```diff
@@ -8,3 +8,21 @@
 * {
   box-sizing: border-box;
 }
+
+.contentEl{
```

Comment:
Please use `.exam2-content` classname, so this style doesn't apply to any other pages.